### PR TITLE
Update grpcio version to 1.41.1

### PR DESCRIPTION
**Pull request by lisyukanna**

### CODE REVIEW

File: controller/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-grpcio==1.20.1
+grpcio==1.41.1
 grpcio-tools==1.20.1
```
Comment (oriyarde) [inline controller/requirements.txt:2]: could we align the grpcio-tools version while we're at it?

Comment (lisyukanna) [reply controller/requirements.txt:2]: what do you mean?

Comment (oriyarde) [reply controller/requirements.txt:2]:
```suggestion
grpcio==1.41.1
grpcio-tools==1.41.1
```

Comment (lisyukanna) [reply controller/requirements.txt]: done

**Review by oriyarde — APPROVED** [review]
LGTM

**Review by matancarmeli7 — APPROVED** [review]
LGTM

**Pull request merged**